app/main/posts/collections/listing.directive.js

```diff
@@ -132,7 +133,12 @@ function CollectionListingController(
     // they will be directed to the collection page
     function goToCollection(collection) {
         $scope.$parent.closeModal();
-        $location.path(collectionsPath + collection.id + '/' + collection.view);
+        var viewParam = collection.view;
```

Comment:
I'm thinking I might not need this at all? Since we have the redirector anyway.. future-self will check because it's EOD